### PR TITLE
GH-210 Add support for SLES 15

### PR DESCRIPTION
**Pull request by ph84172**
I've been running this on a couple of test SLES 15 systems to update the PAM configuration and modify access.conf and it appears to be working correctly.

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -78,7 +79,8 @@
         "9",
         "10",
         "11",
-        "12"
+        "12",
+        "15"
```
Comment (ghoneycutt) [inline metadata.json]: The README will also need to be updated with this info under Limitations at the bottom.

File: data/os/Suse/15.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+pam::common_files_create_links: true
+pam::common_files_suffix:       '_pc'
```
Comment (ghoneycutt) [inline data/os/Suse/15.yaml]: don't align data, just put a single space between the `:` and your data.

```yaml
pam::common_files_suffix: '_pc'
```

**Comment by ghoneycutt** [comment]
@ph84172 looks like you are missing spec fixtures. Take a look at https://github.com/ghoneycutt/puppet-module-pam/tree/master/spec/fixtures

To get these, just copy them from a vanilla system without running puppet on it, then add the header to the top and remove any other comments.

header is

```
# This file is being maintained by Puppet.
# DO NOT EDIT
```

**Comment by ph84172** [comment]
Thanks Garrett, the tests are passing now.  That one failure on the last build looks like a job timing out.

**Comment by ghoneycutt** [comment]
One of the travis jobs timed out, restarting that one.

**Comment by ghoneycutt** [comment]
Fixes #210

**Pull request merged**